conjure/controllers/deploystatus/common.py

```diff
@@ -34,6 +34,7 @@ def wait_for_applications(script, error_cb, msg_cb):
                             result['message']))
                         count += 1
                     continue
+                count = 0
```

Comment:
cosmetic to make sure the current agent status being checked is displayed to the user in the footer